app/code/Magento/InventoryConfigurableProduct/Test/_files/product_configurable.php

```diff
@@ -20,6 +20,9 @@
 
 \Magento\TestFramework\Helper\Bootstrap::getInstance()->reinitialize();
 
+require __DIR__ .
+    '/../../../../../../dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/configurable_attribute.php';
```

Comment:
This may work differently on internal CI.

Created another build to double-check: https://m2build-ur.devops.magento.com/job/All-User-Requested-Tests/23650/